src/styles/components/_chips.scss

```diff
@@ -13,6 +13,7 @@
   flex-flow: row wrap;
 }
 
+.w-post-content .w-chip,
```

Comment:
I think as much as possible we want to avoid descendant selectors. Were you using it to gain more specificity than the `:link` and `:visited` selectors?

Reply:
`.w-post-content` takes higher priority in how `a` tags have underlines. Didn't come up with a better solution atm.

Reply:
ah i see. ok, would you mind adding a comment to mention that.